src/components/stepper/Stepper.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import { Dimensions, LayoutChangeEvent, View } from "react-native";
+// import { useCallback } from "react";
+import { useMemo } from "react";
+import {
+  IOColors,
+  IOSpacer,
+  IOStyles,
+  IOVisualCostants,
+  useIOExperimentalDesign
+} from "../../core";
+
+type StepperProps = {
+  steps: number;
+  currentStep: number;
+  padded?: boolean;
```

Comment:
```suggestion
```
We need to use this component in a fixed position, so we probably always need to add this property. We can take it for granted.